### ISSUE TITLE
Support for TF_KEY_NAME

### ISSUE THREAD

**Issue by GeertJohan** [adammck/terraform-inventory#33]
Hi,

I the `TF_KEY_NAME` override in the source, which is very useful because some servers are only accessable on their private_ip.
Should `TF_KEY_NAME` be documented? And will it stay supported?
Thanks

**Comment by adammck** [comment]
Yes, on both counts. It was added by #28, for the same use-case, but not documented at the time. It will continue to be supported indefinitely, although I'd like to add a `--private` argument to achieve the same thing.

**Comment by simt2** [comment]
It took me some time and debugging to find this hidden feature :)
